dascore/core/spool.py

```diff
@@ -309,6 +316,37 @@ def select(self, **kwargs) -> Self:
         )
         return out
 
+    @compose_docstring(doc=BaseSpool.sort.__doc__)
+    def sort(self, attribute) -> Self:
+        """Sort the Spool based on a specific attribute."""
+        df = self._df
+        inst_df = self._instruction_df
+
+        # make sure we can also cover "time" and "distance" as attributes
+        if attribute == "time":
+            attribute = "time_min"
+        elif attribute == "distance":
+            attribute = "distance_min"
+
```

Comment:
I would like to get away from "time" and "distance" being special so we can support transformed patches that might have differently named dimensions. Let's remove this and add more general logic.

Reply:
Agree.